packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -1516,6 +1516,8 @@ void main() {
   testWidgets('DraggableScrollableSheet controller can be changed', (WidgetTester tester) async {
     final DraggableScrollableController controller1 = DraggableScrollableController();
     final DraggableScrollableController controller2 = DraggableScrollableController();
+    final List<double> loggedSizes = <double>[];
```

Comment:
Can we have two lists, to verify that the event is not sent to the wrong controller?

Reply:
Actually, the old extent will never trigger an event but keep a reference for the `DraggableScrollableController`.